FaqGen/docker/xeon/README.md

```diff
@@ -71,7 +75,7 @@ Since the `compose.yaml` will consume some environment variables, you need to se
 export no_proxy=${your_no_proxy}
 export http_proxy=${your_http_proxy}
 export https_proxy=${your_http_proxy}
-export LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+export LLM_MODEL_ID="meta-llama/Meta-Llama-3-8B-Instruct"
```

Comment:
For gated models, we need to inform users that they must provide a Hugging Face token.  Why not use ‘meta-llama/Meta-Llama-3.1-8B-Instruct' since the new image has already supported?

“For gated models such as [meta-llama/Meta-Llama-3-8B-Instruct"](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct), you will have to pass -e HUGGING_FACE_HUB_TOKEN=<token> to the docker run command above with a valid Hugging Face Hub read token.”

Reply:
> For gated models, we need to inform users that they must provide a Hugging Face token. Why not use ‘meta-llama/Meta-Llama-3.1-8B-Instruct' since the new image has already supported?
> 
> “For gated models such as [meta-llama/Meta-Llama-3-8B-Instruct"](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct), you will have to pass -e HUGGING_FACE_HUB_TOKEN= to the docker run command above with a valid Hugging Face Hub read token.”

For "HUGGING_FACE_HUB_TOKEN", I have add a  reminding for user in readme. In all docker run commands, we have already pass it.
For Llama-3.1, since by now all the validations are done on Llama-3(K8S/performance/accuracy), thus we set Llama-3 as default model currently. After refresh all the testing to Llama3.1 later, we can update it.